.gitignore

```diff
@@ -9,4 +9,5 @@ compile_commands.json
 CTestTestfile.cmake
 _deps
 exec
-calculatorSessionDb.db
```

Comment:
remember to add "new lines" after the last line in every file

Reply:
Added new lines in the end